### ISSUE TITLE
add-migration does not work after modifying Database table

### ISSUE THREAD

**Issue by ghisbo** [dotnet/efcore#23221]
Scaffolding works fine ( DBContext and Models are build based on Database table ( AZ SQLServer ) 
hen adding a field to the table the add-migration executes but the context nor the model is changed!! There is a migration folder with migration in it, but the new field is not added. 
I ran the add-migration with PM and CLI and it does not work in both cases. 
Deleting a field from the table did not work neither. 
FYI: i added and deleted the last field ( test and test2 in the table Vat)   

.....

CREATE TABLE [dbo].[Vat] (
    [Id]             BIGINT     NOT NULL,
    [VatNumber]      CHAR (40)  NOT NULL,
    [CountryCode]    CHAR (10)  NULL,
    [Number]         NCHAR (30) NULL,
    [ValidationDate] DATE       NULL,
    [IsCompanyHash]  BIT        NOT NULL,
    [test]           NCHAR (10) NULL,
    [test2]          NCHAR (10) NULL,
    CONSTRAINT [PK_Vat] PRIMARY KEY CLUSTERED ([Id] ASC)
);

GO
CREATE NONCLUSTERED INDEX [IX_VatNumber]
    ON [dbo].[Vat]([VatNumber] ASC);

GO
EXECUTE sp_addextendedproperty @name = N'MS_Description', @value = N'VatNumber for european companise, GatHash(Company Name) voor buiten eurpa', @level0type = N'SCHEMA', @level0name = N'dbo', @level1type = N'TABLE', @level1name = N'Vat', @level2type = N'COLUMN', @level2name = N'VatNumber';

....

[testConsoleEfCore.zip](https://github.com/dotnet/efcore/files/5500628/testConsoleEfCore.zip)

EF Core version:
Database provider: (e.g. Microsoft.EntityFrameworkCore.SqlServer)
Target framework:  .NET 5.0 RC2
Operating system: Win10 64
IDE:  Visual Studio 2019 16.8

**Comment by ErikEJ** [comment]
Migrations are based on changes to your code, not changes to your database table.

**Comment by ghisbo** [comment]
Ok, I interpreted it incorrectly. thanks

**Comment by ErikEJ** [comment]
If you want to update your code after database table changes, run scaffolding again with Force option, or for an even better experience, use EF Core Power Tools 😁

**Comment by ghisbo** [comment]
Hello
Thanks fort he feedback.
I downloaded it and ran the scaffolding from there. Perfect.
I tried ta change the model and use the migration ( preview ) but nothing happens ??

Best regards

Ghislain

Van: Erik Ejlskov Jensen <notifications@github.com>
Verzonden: vrijdag 6 november 2020 11:59
Aan: dotnet/efcore <efcore@noreply.github.com>
CC: Ghislain Borremans <ghislain@beness.eu>; State change <state_change@noreply.github.com>
Onderwerp: Re: [dotnet/efcore] add-migration does not work after modifying Database table (#23221)


If you want to update your code after database table changes, run scaffolding again with Force option, or for an even better experience, use EF Core Power Tools 😁
